metadata/metadata.ts

```diff
@@ -5679,4 +5678,20 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
       { trait_type: "Tradable", value: "No" },
     ],
   },
+  Walrus: {
+    description:
+      "With his trusty tusks and love for the deep, he'll ensure you reel in an extra fish every time",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/question_mark.png",
+    attributes: [],
+  },
+  Alba: {
+    description:
+      "With her keen instincts, she ensures you get a little extra splash in your catch. +0.5 fish!",
```

Comment:
Adam stated explicitly in one of the streams that a partial fish doesn't really work well.  For buds, that means that the impact is a chance of getting +1 fish.  Should this be changed to align? (50% chance of +1?)